docs/api.md

```diff
@@ -29,9 +29,10 @@
 
 ## Events
 
-| Event             | Type                                  |
-|-------------------|---------------------------------------|
-| `dropdownToggled` | `CustomEvent<{ expanded: boolean; }>` |
+| Event                       | Type                                  | Description                                 |
+|-----------------------------|---------------------------------------|---------------------------------------------|
+| `auroDropdown-triggerClick` | `CustomEvent<any>`                    | Notifies that the trigger has been clicked. |
+| `dropdownToggled`           | `CustomEvent<{ expanded: boolean; }>` |                                             |
```

Comment:
This event is being picked up in the manifest, but there is no documentation to support this? 

There should be an `@fires` line item for this.

Reply:
`dropdownToggled` is not part of this PR. Event also needs to be renamed to our new standard. Ticket #122 opened for this work.